src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -276,22 +278,28 @@ export class IndexPatternsService {
     title: string,
     options: GetFieldsOptions
   ) => {
+    // eslint-disable-next-line no-console
+    console.log('refreshFieldSpecMap start');
     const scriptdFields = Object.values(fields).filter((field) => field.scripted);
     try {
       const newFields = await this.getFieldsForWildcard(options);
       return this.fieldArrayToMap([...newFields, ...scriptdFields]);
     } catch (err) {
+      // eslint-disable-next-line no-console
+      console.log('refreshFieldSpecMap ERROR');
       if (err instanceof IndexPatternMissingIndices) {
         this.onNotification({ title: (err as any).message, color: 'danger', iconType: 'alert' });
         return {};
       }
+      const msg = i18n.translate('data.indexPatterns.fetchFieldErrorTitle', {
+        defaultMessage: 'Error fetching fields for index pattern {title} (ID: {id})',
+        values: { id, title },
+      });
 
       this.onError(err, {
```

Comment:
@mattkime I wonder what happens with this error message? because since the error is no instanced `IndexPatternMissingIndices` this error should be triggered, so the test should fail here and giving an better error message in the screenshot and not the follow up message that is misleading